optaweb-tsp-ui/src/components/LocationList.tsx

```diff
@@ -59,14 +60,18 @@ const renderLocationList = ({
           maxDistance={maxDistance}
           distance={parseInt(distance, 10) || maxDistance}
         />
-        <hr />
+        <br />
+        <Button type="button" style={{ width: '100%' }} onClick={clearHandler}>
+          Clear
+        </Button>
+        <br />
       </CardHeader>
       <CardBody>
         {/*
-               The calculated maxHeight is a hack because the constant 116px depends
-               on the height of Distance and Locations rows (above) and individual location rows.
-               */}
-        <div style={{ maxHeight: 'calc(100vh - 195px)', overflowY: 'auto' }}>
+        The calculated maxHeight is a hack because the last constant depends
+        on the height of CardHeader (above).
+        */}
+        <div style={{ maxHeight: 'calc(100vh - 24px - 24px - 8px - 196px)', overflowY: 'auto' }}>
```

Comment:
I think from maintenance perspective it's definitely better to keep CSS style inline because that means less files to manage. The components that we use now are likely to be replaced soon.